src/main/java/org/docstr/gradle/plugins/gwt/AbstractGwtCompile.java

```diff
@@ -51,6 +52,11 @@ protected void addArgs() {
     argIfEnabled(getDisableGeneratingOnShards(), "-XdisableGeneratingOnShards");
 
     argIfSet("-optimize", getOptimize());
+
+    for (String property: getSetProperties()) {
```

Comment:
Is it possible that `getSetProperties()` is `null`?

Reply:
Looks like it's not as the setter won't be called if no such configuration exists